modules/administration/pages/tshoot-registerclones.adoc

```diff
@@ -19,20 +19,28 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 . Last step
 ////
 
-Sometimes a cloned client (either traditional or Salt) will use the same machine ID as the system they are a clone of.
-This results in {productname} only recognizing one system, rather than two different systems.
-This can be resolved by changing the machine ID of the  cloned system, so that {productname} recognizes them as two different clients.
+
+If you are using {productname} to manage virtual machines, you might find it useful to create clones of your VMs.
+A clone is a VM that uses a primary disk that is an exact copy of an existing disk.
+
+While cloning VMs can save you a lot of time, the duplicated identifying information on the disk can sometimes cause problems.
+
+If you have a client that is already registered, you create a clone of that client, and then try and register the clone, you probably want {productname} to register them as two separate clients.
+However, if the machine ID in both the original client and the clone is the same, {productname} will register both clients as one system, and the existing client data will be over-written with that of the clone.
+
+This can be resolved by changing the machine ID of the clone, so that {productname} recognizes them as two different clients.
 
 [NOTE]
 ====
-Each step of this procedure is performed on the cloned system.
-This procedure does not manipulate the original system, which will still be registered to {productname}.
-The cloned virtual machine must have a different UUID than the original (the UUID is generated by your hypervisor), otherwise {productname} will override the original system data with the new data.
+Each step of this procedure is performed on the cloned client.
+This procedure does not manipulate the original client, which will still be registered to {productname}.
 ====
 
+
+
 .Procedure: Resolving Duplicate Machine IDs in Cloned Salt Clients
 
-. For {slsa}{nbsp}12: If your machines have the same machine ID, delete the file on each client and re-create it:
+. For {slsa}{nbsp}12: If your machines have the same machine ID, delete the file on each duplicated client and re-create it:
```

Comment:
This is about SLE 12, 15, openSUSE, RHEL 7, 8, Ubuntus...

We should rather write "distributions that support systemd"